crypto/cipher/aead_test.cc

```
@@ -446,6 +450,466 @@ TEST_P(PerAEADTest, TestVectorScatterGather) {
+
+  // 2 chunks.
+  ideas.insert({0});
+  ideas.insert({unaligned_at_start});
+  ideas.insert({second_block_start});
+  ideas.insert({last_block_start});
+  ideas.insert({unaligned_at_end});
+  ideas.insert({length});
+
+  // 3 chunks.
+
+  // Only one nonempty chunk.
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)